lightning/src/ln/channelmanager.rs

```diff
@@ -8379,6 +8383,12 @@ macro_rules! create_refund_builder { ($self: ident, $builder: ty) => {
 	}
 } }
 
+/// Defines the maximum number of [`OffersMessage`] to be sent along different reply paths.
+/// Sending multiple invoices increases the chances of successful delivery in case some
```

Comment:
s/invoices/requests

The example given here is for when sending invoice requests (i.e. paying an offer), and thus receiving an invoice on the reply path. The other use is for requesting a refund payment (i.e., sending *invoices*), but that doesn't involve payment id since it is an inbound payment.

Reply:
That makes sense!
The use of the word requests fits more aptly here!
Fixed it in [0354d8d](https://github.com/lightningdevkit/rust-lightning/commit/0354d8dd8ad2608400be58c4de6e5d6c037df5af)

Thanks a lot for the pointer!